kano_feedback/DataSender.py

```diff
@@ -52,6 +52,7 @@ def send_data(text, full_info, subject="", network_send=True):
         full_info (bool): Whether to attach all logs to the payload
         subject (str): The title of the email when sending the logs
         network_send (bool): Whether to send the data to our servers
+        logs_path (str): Path to an existing logs archive to use instead
```

Comment:
Something to think about moving forward, should optional parameters be marked as such in the docstrings?